app/classes/Controllers/ComponentListing.php

```diff
@@ -14,15 +14,33 @@ class ComponentListing extends BaseController {
 
 	public function get() {
 
-		$viewdata = array();
+		$viewdata = array(
+				'primitives' => array(
+					'title' => 'Primitives',
+					'modules' => array(),
+				),
+				'component' => array(
+					'title' => 'Components',
+					'modules' => array(),
+				),
+				'templates' => array(
```

Comment:
I'm not sure "templates" is a good name for these. (I know, I know, naming things is hard!). I wonder if there is a better one that doesn't make it sound like these are HTML-y things...
I guess the only thing in our templates category is o-tech-docs right now?